src/Header/NavDesktop.tsx

```diff
@@ -0,0 +1,50 @@
+import { styled } from '@storybook/theming';
+import React, { FC } from 'react';
+import { useHeaderContext } from './HeaderContext';
+import * as NavigationMenu from '@radix-ui/react-navigation-menu';
+import { NavDesktopLink } from './NavDesktopLink';
+import { NavDesktopContent } from './NavDesktopContent';
+
+const NavigationMenuRoot = styled(NavigationMenu.Root)`
+  position: relative;
+  display: flex;
+  justify-content: center;
+  z-index: 1;
+`;
+
+const NavigationMenuList = styled(NavigationMenu.List)`
+  display: flex;
+  justify-content: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  gap: 9px;
+`;
+
+const NavigationMenuItem = styled(NavigationMenu.Item)`
+  position: relative;
+`;
+
+export const NavDesktop: FC = () => {
+  const { navDesktop, setActive } = useHeaderContext();
+
+  return (
+    <NavigationMenuRoot>
+      <NavigationMenuList>
+        {navDesktop &&
+          navDesktop.map((item) => (
```

Comment:
@winkerVSbecks If that make more sense for you to follow Radix structure I can do that. What I tried to do is merge Trigger and Link into one since they share the same design and put a condition on Content when it's there.

Reply:
IMO it's worth splitting link and trigger

Reply:
Ok I'll refactor the code.

Reply:
Done